sre-operational-fabric.md

```diff
@@ -0,0 +1,119 @@
+# (Remote) SRE - Rackspace Private Cloud
+
+## Role
+
+Site Reliability Engineering (SRE) is an engineering discipline that combines
+software and systems engineering to build and run large-scale, massively
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
```

Comment:
The sentences after the first one come off as if they're barked, or somewhat of a bulleted list in paragraph form. I think this should start off as any of "A Site Reliability Engineer", "An SRE", or "SREs ensure".

Reply:
I would soften the "Act as..." and "Focus on..." sentences below to start with stuff like "You will" or something like that.

Reply:
It's talking about the discipline not the person. I fixed the second bit though.